src/pykiso/cli.py

```diff
@@ -204,6 +204,19 @@ def eval_user_tags(click_context: click.Context) -> Dict[str, List[str]]:
     default=True,
     help="default, test results are only displayed in the console",
 )
+@click.option(
+    "--step-report",
```

Comment:
wouldn't it be easier to have just one flag ? 
--step-report    -> creates report with default name
--step-report custom.html  - > report with custom name

Reply:
Just test if the path to a report is given then create it otherwise consider that no report is mandatory

Reply:
I implement it like @BKaDamien suggested. @Pog3k solution does not work because it is not supported by Click 7.0.x which is required by auto-changelog.